### DISCUSSION TITLE
Tuya integration

### DISCUSSION THREAD

**Discussion by 88Marius88** [indomus/forum#998] [GENERALE - Discussioni generiche]
Dopo l'ultimo aggiornamento non riesco piu a collegare accaunt tuya e smartilyfe. Qualcuno e riuscito? accedendo a quella pagina di tuya iot. ?

**Comment by befaleg** [comment]
anche io avevo lo stesso problema, ho risolto aprendo un nuovo account sulla pagina iot (abbandonando il vecchio account) 
 per poi agganziarla all'app e poi ancora interìgrarla in ha... ora è tornato a funzionare tutto

**Reply by 88Marius88** [reply]
Che usi per integrarla tuya v2 o tuya normale? Come account invece sullo smartphone cosa usi?

**Reply by Newreader** [reply]
Per far funzionare l'integrazione Tuya devi semplicemente (!) modificare la password legata all'app tuya o smart life (possibilmente inserendo solo numeri e lettere). IoT non lo devi toccare. Dopo aver fatto questo lanci nuovamente l'integrazione Tuya che ha questo punto si dovrebbe attivare. Fammi sapere.

**Reply by 88Marius88** [reply]
Ho modificato la password ho ricaricato l'integrazione tuya ed ha funzionato magicamente hahahaha!!!! Ste cose cosi ti fanno imbestialire! Grazie mille!

**Reply by Newreader** [reply]
Figurati. Comunque la soluzione è scritta in questo issue:
https://github.com/tuya/tuya-home-assistant/issues/663
e come tale probabilmente verrà eliminato.
Comunque al momento lo evidenzio a beneficio di quant'altri ne avessero bisogno.

**Comment by nromagna** [comment]
Ciao a tutti, aggiornato a **core-2021.11.1** ed avuto qualche problema con Tuya... 
Elimianto la vecchia integrazione ed aggiunta quella nuova tramite il **https://iot.tuya.com/** ma ho riscontrato grossi problemi nell'abbianre il Device nell'ultimo passaggio. Così ho cfreato un nuovo account nell'app SmartLife ed sono riuscito ad integrarlo. Ovviamente essendo un nuovo profilo senza nessun dispositivo dentro.
Ho provato ad entrare nel vecchio profilo (seguendo un po' la scia del Sonoff) e condividendo i due dispositivi che avevo con il nuovo profilo che sono stati ovviamente visti dall'app ma **NON da HA.**
Quindi chiedo, devo eliminare il vecchio profilo e RI-abbinare i due dispositivi con il nuovo profilo oppure ho sbagliato io qualcosa?

L'errore al momento dell'abbinamento dispositivo tramite qrcode del iot era: "impossibile eseguire la scansione del codice qr per aggiungere un dispositivo già in uso in un altro centro dati"

Grazie a tutti.

**Reply by nromagna** [reply]
Come NON detto... rifatto la MEDESIMA procedura di questa mattina ed è partito tutto... mah....

**Reply by Franco2255** [reply]
Ciao ragazzi o un problema con integrazione Tuya, dove ricaricare il servizio tutti i giorni altrimenti non aggiorna lo stato degli  switch.
Qualcuno ha lo stesso problema, come posso risolvere?
Grazie

**Reply by Newreader** [reply]
Leggi qui:
https://github.com/home-assistant/core/pull/62858
Dovresti in buona sostanza caricare almeno la versione 2021.12.6 di HA per risolvere il problema (mi sembra che l'attuale sia la 2021.12.8 che ovviamente va benissimo)

**Reply by Franco2255** [reply]
Grazie Newreader,
ho la versione 2021.12.8 ieri ho ricaricato il sevizio e per ora va vediamo quanto regge.

**Reply by lelefor** [reply]
Io continuo ad avere il problema che dopo 1 o 2 giorni HA non aggiorna più  i sensori tuya.  Qualcuno ha una soluzione definitiva?
